evm/src/executor/inspector/cheatcodes/mod.rs

```diff
@@ -498,6 +502,31 @@ where
             ])
         }
 
+        // Record writes with sstore (and sha3) if `StartMappingRecording` has been called
+        if let Some(mapping_slots) = &mut self.mapping_slots {
+            match interpreter.contract.bytecode.bytecode()[interpreter.program_counter()] {
+                opcode::SHA3 => {
+                    if interpreter.stack.peek(1) == Ok(0x40.into()) {
+                        let address = interpreter.contract.address;
+                        let offset = interpreter.stack.peek(0).expect("stack size > 1").as_usize();
+                        let low: U256 = interpreter.memory.get_slice(offset, 0x20).into();
+                        let high: U256 = interpreter.memory.get_slice(offset + 0x20, 0x20).into();
+                        let result: U256 = keccak256(interpreter.memory.get_slice(offset, 0x40)).into();
```

Comment:
the `.into()` conversion will no longer work once we transition to revm 3.0 and ruint types (U256).

I'd like to use `U256::from` here instead

Reply:
Is there any difference between `U256::from` and `.into()`?
According to the [doc](https://doc.rust-lang.org/std/convert/trait.Into.html),
* `From<T> for U` implies `Into<U> for T`

Reply:
yes, but this currently does not work for https://github.com/paradigmxyz/uint because of const generics and the `TryFrom` trait (which is a bit weird, can't find the explainer rn but there are issues with that trait)

so ruint uses `fn from`

but yes, usually they are the same